### DISCUSSION TITLE
v2.4.8

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#135] [Announcements]
- New Providers Animerulz and MoviesApi
- Other Providers improved (Multistream, World4ufree, PrimeWire)
- Bug fixes and Performance Improvement
- Haptics

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.4.8'>v2.4.8</a>.</em>

**Comment by FearlessPs** [comment]
Dooflix not working

**Reply by Zenda-Cross** [reply]
what's the problem, working for me
if home page not loading try refreshing

**Comment by Stormsprit** [comment]
Hlw bro i want to ask why this app flag as malacious on virus total any reason

**Reply by Zenda-Cross** [reply]
I checked its 1/66. I would say it's a false positive. Google Play Protect also indicates it's safe, and the source code is available for you to build from source.

**Reply by Zenda-Cross** [reply]
The issue has been resolved in the most recent version. You can verify that it is no longer being flagged as malicious.

**Reply by Stormsprit** [reply]
Yaaa thnks
